src/lib/WalletStorage.ts

```diff
@@ -58,6 +82,10 @@ function createLocalStorageInterface(): IWalletStorage {
  * Exposes methods that allows to manipulate or watch the device storage.
  * Allows also to synchronize the state between the app and the extension background.
  */
-export const WalletStorage: IWalletStorage = (!RUNNING_IN_TESTS && IS_EXTENSION && browser)
+export const WalletStorage: IWalletStorage = (
+  !RUNNING_IN_TESTS
+  && !IS_OFFSCREEN_TAB
+  && IS_EXTENSION_BACKGROUND
+  && browser)
```

Comment:
If the condition is multiline please put the ending bracket in new line and add indentation to the statements.